packages/reports/addon/components/filter-collection.ts

```diff
@@ -26,6 +30,8 @@ type FilterItem = {
 export default class FilterCollection extends Component<FilterCollectionArgs> {
   @service requestConstrainer!: RequestConstrainer;
 
+  componentElement!: HTMLElement;
```

Comment:
Can we rename this to lastAddedFilterElement or something similar? I feel like we typically use componentElement to mean the top level element in the template
```suggestion
  componentElement: HTMLElement | undefined;
```